src/Abbyy.CloudSdk.V2.Client/Abbyy.CloudSdk.V2.Client.csproj

```diff
@@ -1,10 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net472;net461;net45;netstandard2.0</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <LangVersion>7.1</LangVersion>
-    <Version>1.0.1</Version>
+    <AssemblySemVer>1.0.1</AssemblySemVer>
```

Comment:
Why 1.0.1 ? :) It will overwrite on build?

Reply:
Yes, it is a default value and build sets it